### ISSUE TITLE
Send 'gci server build' build logs to client

### ISSUE THREAD

**Issue by arschles** [arschles/gocons#43]
After #35 is done. Need to figure out how to send logs down along with the completed build artifact. [Multipart responses](https://www.w3.org/Protocols/rfc1341/7_2_Multipart.html) may work for this

**Comment by arschles** [comment]
this issue only applies to when a gci server is implemented. may be necessary to split this out into another repo. when implemented, may be useful to use gRPC for the req/response cycle
